app/assets/stylesheets/components/layout.scss

```diff
@@ -189,3 +189,14 @@ i.fa {
     border-right: 1px solid $primary-colour;
   }
 }
+
+//popover css used in signup page for client side validation
+
+#signup_page .popover{
+    background-color: rgba(0,0,0,0.6);

```

Comment:
That's what I thought, these styling can be removed then. 

I didn't see any form validation using popover. It's using the built-in validators. I think you can use [setCustomValidity](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/setCustomValidity) method to achieve the same look and feel.

What do you think, @mssola?